packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -172,6 +173,44 @@ class TransferRootsDb extends BaseDb {
       )
     })
   }
+
+  async getUnsettledTransferRoots (
```

Comment:
Now that we have this in the TransferRootsDb, I believe we should remove `getUnsettledBondedWithdrawalTransfers()` from the TransfersDb.

Reply:
good idea, will remove it